content/javascript/concepts/window/terms/clearTimeout/clearTimeout.md

```diff
@@ -0,0 +1,39 @@
+---
+Title: 'clearTimeout()'
+Description: 'Clears a previously scheduled timeout function.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Development'
+Tags:
+  - 'Arguments'
+  - 'Functions'
+  - 'Parameters'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**`clearTimeout()`** is a window method that stops a previously scheduled [`setTimeout()`](https://www.codecademy.com/resources/docs/javascript/window/setTimeout) from running.
+
+## Syntax
+
+`clearTimeout()` receives a timeout ID as its argument, which is returned by the corresponding call of `setTimeout()`.
```

Comment:
There are probably some entries that have sentences before the syntax, but our format is generally to show the syntax and then include information about the parameters, returns, etc.

Reply:
@caupolicandiaz I made some changes from what I thought you were suggesting. Let me know if it's better now.